storage/mysql/log_admin.go

```diff
@@ -0,0 +1,82 @@
+package mysql
+
+import (
+	"fmt"
+
+	"github.com/google/trillian/crypto"
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/storage/cache"
+)
+
+const (
+	setTreePropertiesSQL = `INSERT INTO Trees(TreeId,KeyId,TreeType,LeafHasherType,TreeHasherType,AllowsDuplicateLeaves) VALUES(?, ?, ?, ?, ?, ?)`
+	setTreeParametersSQL = `INSERT INTO TreeControl(TreeId,ReadOnlyRequests,SigningEnabled,SequencingEnabled,SequenceIntervalSeconds,SignIntervalSeconds) 
+		VALUES(?, ?, ?, ?, ?, ?)`
+	deleteTreeSQL        = `DELETE FROM Trees WHERE TreeId = ?`
+	deleteTreeControlSQL = `DELETE FROM TreeControl WHERE TreeId = ?`
+
+	keyID                = 1
+	treeType             = "LOG"
+	leafHasherType       = "SHA256"
+	treeHasherType       = "SHA256"
+	allowDuplicateLeaves = false
+	readOnly             = false
+	signingEnabled       = false
+	sequencingEnabled    = false
+	sequenceInterval     = 1
+	signInterval         = 1
+)
+
+// CreateTree instantiates a new log with default parameters.
+// TODO(codinglama): Move to admin API when the admin API is created.
+func CreateTree(treeID int64, dbURL string) error {
+	th := merkle.NewRFC6962TreeHasher(crypto.NewSHA256())
```

Comment:
May need the ritual TODO on supporting other hash schemes but if it's purely temporary don't worry too much.